pkg/frontend/querymiddleware/roundtrip.go

```diff
@@ -310,10 +311,21 @@ func isInstantQuery(path string) bool {
 
 func defaultInstantQueryParamsRoundTripper(next http.RoundTripper, now func() time.Time) http.RoundTripper {
```

Comment:
[unrelated from this PR, but...] I just noticed `now` is unused. Shouldn't we use it instead of `time.Now()`? Or remove the param.

Reply:
Removed here: https://github.com/grafana/mimir/pull/3588/commits/d08c6ae6ddaa7c9b5e75ceeb719de8eeedd0c999